tests/test_pack_sim.py

```diff
@@ -35,6 +35,9 @@ def test_main(tmp_path, mocker):
     assert Path("include/reek.pvt").exists()
     assert Path("include/swof.inc").exists()
 
+    # Test that comment is process properly after INCLUDE keyword
+    assert "INCLUDE\nINCLUDE" not in Path(ECLCASE).read_text(encoding="utf8")
+
```

Comment:
Nice, love a simple fix 😄 however, I don't think this addition to the test catches if it fixes it or not because I don't think the reek model has the type of comment that causes the problem. Consider adding a separate regression test case for this-- something like this perhaps should do the trick (this might not be totally right):

```python
def test_double_include_before_comment(tmp_path, mocker):
     os.chdir(tmp_path)

     some_include = "some.inc"
     with open(some_include, "w", encoding="utf-8") as fout:
         fout.write("-- Comment")

     data_file = "TMP.DATA"
     with open(data_file, "w", encoding="utf-8") as fout:
         fout.write(f"INCLUDE\n--Comment\n  '{some_include}' /")
     mocker.patch("sys.argv", ["pack_sim", data_file, "out/"])
     pack_sim.main()

     # Test that comment is process properly after INCLUDE keyword
     assert "INCLUDE\nINCLUDE" not in Path(f"out/{data_file}").read_text(encoding="utf8")
```

Reply:
That's silly of me 😜 😆  . I made changes to reek model to trigger the bug, but didn't stage it. But I like your test better, so I will include it instead. 👍